ygnmi/ygnmi_test.go

```diff
@@ -2226,6 +2226,138 @@ func TestDelete(t *testing.T) {
 	}
 }
 
+func TestBatchGet(t *testing.T) {
+	fakeGNMI, c := newClient(t)
+	aLeafStatePath := testutil.GNMIPath(t, "/remote-container/state/a-leaf")
+	twoPath := testutil.GNMIPath(t, "/parent/child/state/two")
+	aLeafSubath := testutil.GNMIPath(t, "/remote-container/*/a-leaf")
+
+	tests := []struct {
+		desc                 string
+		stub                 func(s *testutil.Stubber)
+		config               bool
+		paths                []ygnmi.PathStruct
+		wantSubscriptionPath []*gpb.Path
+		wantVal              *ygnmi.Value[*exampleoc.Root]
+		wantErr              string
+	}{{
+		desc: "state leaves",
+		stub: func(s *testutil.Stubber) {
+			s.Notification(&gpb.Notification{
+				Timestamp: 100,
+				Update: []*gpb.Update{{
+					Path: aLeafStatePath,
+					Val:  &gpb.TypedValue{Value: &gpb.TypedValue_StringVal{StringVal: "foo"}},
+				}, {
+					Path: twoPath,
+					Val:  &gpb.TypedValue{Value: &gpb.TypedValue_StringVal{StringVal: "bar"}},
+				}},
+			}).Sync()
+		},
+		paths: []ygnmi.PathStruct{
+			root.New().RemoteContainer().ALeaf(),
+			root.New().Parent().Child().Two(),
+		},
```

Comment:
i don't understand what you mean "the config path returned for aLeafConfigPath right?"

a path structs represents exactly one path
root.New().RemoteContainer().ALeaf() -> "/remote-container/*/a-leaf"
root.New().Parent().Child().Two() -> "/parent/child/state/two"

Reply:
Sorry for the lack of the clarity, I meant that right now we're returning `aLeafStatePath` and `twoPath`. Since we have a wildcard path should we also return an `aLeafConfigPath` in the Notification list?

Reply:
added